src/posh-git.psd1

```diff
@@ -1,10 +1,10 @@
 @{
 
 # Script module or binary module file associated with this manifest.
-ModuleToProcess = 'posh-git.psm1'
+RootModule = 'posh-git.psm1'
 
 # Version number of this module.
-ModuleVersion = '0.7.0'
+ModuleVersion = '1.0.0'
```

Comment:
I am mostly concerned with the issue template version script showing a difference between loading from `develop`/`master` and from Chocolatey or the Gallery.

An alternative would be to include the posh-git module's path, perhaps with a replacement of `$Home` with `~`?